stargate-starter/src/main/java/io/stargate/starter/Starter.java

```diff
@@ -125,6 +123,16 @@
     @Option(name = {"--emulate-dbaas-defaults"}, description = "Updated defaults reflect those of DataStax Astra at the time of the currently used DSE release")
     private boolean emulateDbaasDefaults = false;
 
+    @Order(value = 14)
+    @Option(name = {"--developer-mode"}, description = "Defines whether the stargate node should also behave as a " +
+        "regular node, joining the ring with tokens assigned in order to facilitate getting started quickly and not " +
+        "requiring additional nodes or existing cluster")
+    boolean developerMode = false;
+
+    @Order(value = 15)
+    @Option(name = {"--bind-to-listen-address"}, description = "When set, it binds web services to listen address only")
```

Comment:
Please add these to starctl as well since we also support env vars.

Reply:
Good catch, I've added them.